Ethics Book: Self Reflection Chapter (#2169)
alan-turing-institute/the-turing-way: book/website/ethical-research/self-reflection/sr-prompts.md

(er-self-reflection-promts)=
# Self Reflection Prompts

Reflexivity is the process of using self-reflection to understand ourselves and our beliefs, and how these influence how we create knowledge through our work in data science. 
There are lots of different aspects you could consider, and the questions below are some prompts to help get you started. 

## Questions about my own identity 

> - What is considered ethical in my culture, that other cultures may see differently?

Reviewer: ```suggestion
- What is considered ethical in my culture that other cultures may see differently?
```

- What are my own {ref}`Social Graces<er-self-reflection-social-graces>`? 
- How have my Social Graces shaped my experiences of life? 
- How have my Social Graces shaped the decisions that have led me to do what I do now?
- What are the Social Graces of my family and friends? 

>   - Are there Graces where we vary a lot/little?

Reviewer: ```suggestion
  - Are there Social Graces where we vary a lot/little?
```

- How do the people who work in my field represent the people our work serves?
  - How do I relate to the people I am trying to serve with my work? 

## Questions about my relationship to knowledge

- What types of knowledge do I value? 
- Am I more convinced by a certain type of data over another - why is that?
- Are there certain people I am more likely to believe, or find convincing? 
- What type of data presentation do I find convincing? 
- What type of statistics do I view as more robust (and why)? 
- What checks do I think data and analysis need to go through to be considered robust? 
- What are my views on qualitative data vs. quantitative data?

## Questions about my relationship to my research topic

- What do I believe are 'truths' about the concept I study?
    - What evidence do I base those truths on? Who developed this evidence?
- Who do I most commonly cite in my research area, and how are their perspectives similar or different to mine?
-  Whose perspectives are important in answering this research topic?
- Am I the best person to answer this question - what might I be missing?

## Questions about my data

> - Who isn't included in the data I have collected or am using second-hand?

Reviewer: ```suggestion
- Who isn't included in the data I have collected or reusing?
```
